components/toolbar/Toolbar.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+interface Props {
+    children: React.ReactNode;
+}
+
+const Toolbar = ({ children }: Props) => <div className="toolbar flex noprint">{children}</div>;
```

Comment:
We don't have those in the ContactToolbar, @nico3333fr . Should we actually have them?

Reply:
Don't have either.

Reply:
Yes, this is not absolutely mandatory for Contacts and Mail, but there will be no impact of adding them: basically, this is more a bullet-proof version in case of too many content inside the toolbar (which can happen when there is text, can be text-zoomed, etc.).